src/test/ui/lint/lint-exceeding-bitshifts.rs

```diff
@@ -1,12 +1,12 @@
+// check-pass
 // revisions: noopt opt opt_with_overflow_checks
+// compile-flags: --emit=link
```

Comment:
I think this makes it equivalent to `build-pass`; what is the point of this flag?

Reply:
Yeah, not sure what I needed it for. I've reverted the flags back to the way they were before.